tests/test_pipeline/conftest.py

```diff
@@ -168,7 +168,7 @@ def step_ts() -> Tuple[TSDataset, pd.DataFrame, pd.DataFrame]:
     forecast_df.columns = pd.MultiIndex.from_product(
         [[segment], ["target", "fold_number"]], names=("segment", "feature")
     )
-
+    forecast_df.sort_index(axis=1, inplace=True)
```

Comment:
Can we fix it above at 168 line by hand changing `target` and `fold_number` to make it more determinitic?